src/encoded/static/components/app.js

```diff
@@ -297,6 +298,27 @@ export default class App extends React.PureComponent {
             });
         }
 
+        if (statusMessagesURL) {
+            ajax.load(
+                statusMessagesURL,
+                function(res){
+                    const { events = [] } = res;
+                    console.info(`Received ${events.length} events from "${statusMessagesURL}".`);
+                    events.forEach(function({ name: title, message, style = "warning", navigateDisappearThreshold = 3 }){
+                        Alerts.queue({ title, message, style, navigateDisappearThreshold });
+                    });
+                },
+                "GET",
+                null,
+                function(err){
+                    console.error("Error getting status events", err);
+                },
+                // Clear headers for CORS
+                {}, //{ "Cache-Control": "max-age=120" }
+                ["Accept", "Content-Type", "Authorization", "X-Requested-With"]
```

Comment:
At @alexkb0009's suggestion, I added more headers that may fix the need for some of this. This is from the current source on the lambda function (not yet in source control, just on AWS's console). These headers are added to all responses from that endpoint (the one you call `statusMessagesURL` here)
```
CORS_HEADERS = {
    # It may be that only GET is needed, but just in case. -kmp&akb 20-Mar-2020
    "Access-Control-Allow-Methods": "GET,HEAD,OPTIONS",
    "Access-Control-Allow-Headers": ",".join([
        "Origin",
        "Accept",
        "X-Requested-With",
        "Content-Type",
        "Access-Control-Request-Method",
        "Access-Control-Request-Headers",
        "Cache-Control",
        "Authorization",
    ]),
    "Access-Control-Allow-Origin": "*",
}
```